doc/Polygon.html

```diff
@@ -64,12 +64,20 @@ <h2 id="boundary">Boundary and Rings</h2>
 			</table>
 
 			<h2>Valid polygons</h2>
-			<p>A valid polygon has no overlapping rings, no self-intersections except possibly
-			at vertices, no dangling segments and, in general, an arbitrary point can always be
-			classified unambiguously as either in the exterior, in the interior or on the
-			boundary of the polygon. A valid polygon is said to be <i>simple</i>.
+			<p>A valid polygon has the following main property:
+				<li>for every polygon segment, the polygon exterior is unambiguously to the left of the segment and
+					the polygon interior is unambiguously to the right of the segment.</li>
+				As a result of the above property the following is also true:
+				<li>segments can touch other segments only at the end points,</li>
+				<li>segments have non-zero length,</li>
+				<li>exterior rings are clockwise and holes are counterclockwise,</li>
```

Comment:
Can you change "exterior" to "outer" just because you talk about the polygon exterior above and it might be confusing.

Reply:
Done.